app/src/main/java/com/greenfox/fuchsit/zerdareader/activity/BaseActivity.java

```diff
@@ -0,0 +1,36 @@
+package com.greenfox.fuchsit.zerdareader.activity;
+
+import android.content.SharedPreferences;
+import android.os.Bundle;
+import android.preference.PreferenceManager;
+import android.support.v7.app.AppCompatActivity;
+
+import com.greenfox.fuchsit.zerdareader.R;
+
+public abstract class BaseActivity extends AppCompatActivity {
+
+    SharedPreferences timePreferences;
```

Comment:
@zsofinnemeth this is a default sharedpreferences, not just the time can be stored in here, so name it like that